content/ember-4-6-released.md

```diff
@@ -0,0 +1,174 @@
+
+For more details on changes in Ember.js 4.6, please review the [Ember.js 4.6.0 release page](https://github.com/emberjs/ember.js/releases/tag/v4.6.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data
+
+The Ember Data team has multiple releases to announce! We will go through them
+one by one in the section below.
+
+Firstly, Ember CLI's blueprints for creating new apps and upgrading existing apps
+still point to v4.4 of Ember Data. If you want to take advantage of the progress
+described below, update the version number in your `package.json` after completing
+your upgrade.
+
+Now let's cover what is in v4.5 and v4.6. These two versions are identical.
+The aim is to catch up to the versioning number of Ember CLI and Ember.js.
+
+Version 4.5 includes a number of new deprecations, a significant internal refactoring of the store architecture and several bug fixes described below. 
+
+#### Node support
+
+v4.5 of Ember Data drops support for Node 12. Node 12 became end of life (it no longer receives security updates) in April 2022.
+
+#### Bug Fixes
+
+Ember Data 4.5 introduced 9 bug fixes. Additional Details are in the [release notes](https://github.com/emberjs/data/releases/tag/v4.5.0).
+
+* [#8083](https://github.com/emberjs/data/pull/8083) [BUGFIX] Serialize null array items as empty strings 
+* [#7901](https://github.com/emberjs/data/pull/7901) <!-- alex ignore waiter -->`waiter` should be always on
+* [#8042](https://github.com/emberjs/data/pull/8042) `RecordState` cleanup 
+* [#8008](https://github.com/emberjs/data/pull/8008) schedule `watchTypeIfUnseen` to prevent loop
+* [#8013](https://github.com/emberjs/data/pull/8013) Fix generated import paths for test setup functions in addons
+* [#8025](https://github.com/emberjs/data/pull/8025) Fix `PromiseProxy` Deprecation for Save ([@runspired](https://github.com/runspired))
+* [#7902](https://github.com/emberjs/data/pull/7902) test for async hasMany does not trigger relationship fetch
+* Pass correct args to `keyFor<Attribute|Relationship>` when normalizing
+* [#7945](https://github.com/emberjs/data/pull/7945) fix for an issue with computed chains not updated for async hasMany loading
+
+#### Features
+
+Ember Data 4.5 introduced improvements to build size.
+
+Users of solely the core of Ember Data may expect their builds to be 35-50% smaller. This is due to most of the legacy support for `@ember-data/model` having now found its final location at home in that package instead of within the `@ember-data/store package`. The size of `@ember-data/store` is reduced from ~16kb compressed to ~9.5kb compressed, potentially a little less depending on what deprecations are resolved and what other packages are present. We expect the core to continue to shrink as we complete the final stages of removing `InternalModel`, complete implementations for recently accepted deprecation RFCs, and introduce `RecordDataV2`.
```

Comment:
@runspired can you help scope/qualify this statement?

> Users of solely the core of Ember Data may expect their builds to be 35-50% smaller.

What does this mean? Is it that a specific ember-data package is 35-50% smaller? Thanks!

Reply:
Broadly speaking this means the store package. It currently also means ember-inflector as that's still a required dependency though it being required has been RFCs to deprecate. Notably users of ember-m3 tend to use only the store package.

Reply:
incorporated, thank you